modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java

```diff
@@ -225,10 +225,17 @@ public Number getFadeDuration() {
 
         // Use of builder is easier for code examples; but fills in SLD defaults
         // Currently only applies the opacity.
+        if(getBrightnessMax() != null || getBrightnessMin() != null){
```

Comment:
Is this test looking for both of those to not be null? I assume you're trying to prevent multiplying by null in 231/232. If so, you probably wanted `&&`.

I'd also suggest brackets to make it clearer.

Maybe some more spaces here to be consistent with other code too.